pkg/windows/windows.go

```diff
@@ -507,6 +513,33 @@ func (vm *windows) ConfigureWICD(watchNamespace, wicdKubeconfigContents string)
 	return nil
 }
 
+func (vm *windows) RestoreAWSRoutes() error {
+	serviceExists, err := vm.serviceExists(ec2LaunchV2ServiceName)
+	if err != nil {
+		return fmt.Errorf("error checking if %s service exists: %w", ec2LaunchV2ServiceName, err)
+	}
+	// We don't want ensureServiceIsRunning to create the service if it does not exist, so we return without an error.
+	// Returning an error does not make sense as we could have an AMI configured without th EC2Launch service present
+	// and the route created using some other customer specific method which is unknown to us.
+	if !serviceExists {
+		vm.log.Info("EC2Launch v2 service not present")
```

Comment:
Consider adding the name of the service used in the query i.e. `ec2LaunchV2ServiceName`